content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -0,0 +1,416 @@
+# Pipelines Files (`dvc.yaml`)
+
+You can construct data science or machine learning pipelines by defining
+individual [stages](/doc/command-reference/run) in one or more `dvc.yaml` files
+(or _pipelines files_). Stages form a pipeline when they connect with each other
+(forming a _dependency graph_, see `dvc dag`). Refer to
+[Data Pipelines](/doc/start/data-pipelines).
+
+> Note that a helper command, `dvc run`, is available to create (and execute)
```

Comment:
> OK Dave, I'm back with you.

We're actually trying to encourage people to write dvc.yaml files manually but still need to mention `dvc run` since it's there.

Is your comment based on your previous experience with DVC where `run` was more prevalent, or do you think the text here doesn't convey the intentions I just mentioned?

Reply:
Yes, I think it makes sense in the context of wanting to discourage `dvc run`. I'm not sure it's clear from the note that `dvc run` adds the stage to `dvc.yaml`, but maybe that's unnecessary if we don't really want users relying on it.